### ISSUE TITLE
Converted saves crashing the game after 1.7.36.0 update.

### ISSUE THREAD

**Issue by bagarwa** [HarukaMa/starfield-xgp-import#10]
I have saves from Steam's 1.7.33.0 version of the game. The game got updated to 1.7.36.0 today. I used this converter to convert the existing steam saves to xgp saves. The tool ran fine and it says it converted the saves fine. But when I try to launch the game in xbox app, it goes till loading the game and then crashes. 

Launch the game -> warning and Bethesda logo -> Game's home screen -> Continue -> shows the converted save file -> Ok -> loading screen -> crash. 

Can you please check whether the tool needs some change to support 1.7.36.0 version of the game?

**Comment by HarukaMa** [comment]
I didn't have time to play the game lately, but just to confirm, 1.7.36 game failed to load converted 1.7.33 save file? Does this happen with every save file?

**Comment by bagarwa** [comment]
Yes, 1.7.36 game had failed to load converted 1.7.33 save file. But apparently not every save file. 

When I had created this thread, I had converted only my latest save file which was an Exitsave0 sfs file. When I tried to load xgp converted version of this, the game crashed as mentioned. 

But later I tried converting more of the earlier saves. Autosave1 Autosave2 Autosave3 and Quicksave0 sfs files. I was able to load the xgp converted save of the Quicksave and played fine then on.
